cao_van_thao/laravel_app/example-app/app/Http/Requests/StoreProductRequest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreProductRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required | max: 50',
```

Comment:
```suggestion
            'name' => 'required|max:50',
```
chỗ này thì ko cần dấu cách nhé, ở dưới cũng đương tự